src/backend/distributed/sql/citus--11.2-1--11.3-1.sql

```diff
@@ -2,3 +2,6 @@
 
 -- bump version to 11.3-1
 
+-- we introduce node_tokens, which will be used internally to
+-- limit the number of parallel tasks running per node
+ALTER TABLE pg_catalog.pg_dist_background_task ADD COLUMN node_tokens int[] DEFAULT NULL;
```

Comment:
In this case I think it makes sense not to allow NULL and simply use the empty array as the zero value.